nomenclature/definitions/variable/technology/power-plant.yaml

```
@@ -156,9 +156,9 @@ Number of Units|Electricity|Coal:
 Number of Units|Electricity|Lignite:
    description: Number of units of kind "Lignite power plant "
    unit:
-
-Maximum Active Power|Electricity|Biomass|Traditionnal:
-   description: Maximum active power of one typical Traditionnal Biomass power plant
+   
+Maximum Active power|Electricity|Biomass|Traditional:
```

Suggest minor editorial change to use only capitalized words in variable names
```suggestion
   
Maximum Active Power|Electricity|Biomass|Traditional:
```